Tests/MapboxMapsTests/Annotations/Mocks/MockAnnotationOrchestatorImpl.swift

```diff
@@ -5,24 +5,62 @@ final class MockAnnotationOrchestatorImpl: AnnotationOrchestratorImplProtocol {
     @Stubbed
     var annotationManagersById: [String: AnnotationManager] = [:]
 
-    func makePointAnnotationManager(id: String,
-                                    layerPosition: LayerPosition?,
-                                    clusterOptions: ClusterOptions?) -> AnnotationManagerInternal {
-        fatalError("TODO")
+    struct MakePointAnnotationManagerParams {
+        var id: String
+        var layerPosition: LayerPosition?
+        var clusterOptions: ClusterOptions?
     }
-    func makePolygonAnnotationManager(id: String,
-                                      layerPosition: LayerPosition?) -> AnnotationManagerInternal {
-        fatalError("TODO")
+    let makePointAnnotationManagerStub = Stub<MakePointAnnotationManagerParams, AnnotationManagerInternal>(defaultReturnValue: PointAnnotationManager(id: "test", style: MockStyle(), layerPosition: .default, displayLinkCoordinator: MockDisplayLinkCoordinator(), offsetPointCalculator: .init(mapboxMap: MockMapboxMap())))
```

Comment:
These declarations seems to be mighty long, how about breaking them into multiple lines?
```suggestion
    let makePointAnnotationManagerStub = Stub<MakePointAnnotationManagerParams, AnnotationManagerInternal>(
        defaultReturnValue: PointAnnotationManager(
            id: "test",
            style: MockStyle(),
            layerPosition: .default,
            displayLinkCoordinator: MockDisplayLinkCoordinator(),
            offsetPointCalculator: .init(mapboxMap: MockMapboxMap())
        )
    )
```